src/delta.rs

```diff
@@ -339,6 +365,7 @@ fn handle_hunk_line(
             Painter::paint_lines(
                 vec![syntax_style_sections],
                 vec![vec![(style::NO_BACKGROUND_COLOR_STYLE_MODIFIER, &line)]],
+                line_numbers,
```

Comment:
How about constructing this argument using the `vec!` macro rather than creating the Vec and then pushing to it?

Reply:
Done